### PR TITLE
Remove old time deprecated repo.clickhouse.com from docs

### PR DESCRIPTION
**Pull request by Felixoid**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Stop confusing people with repo.clickhouse.com

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

<details>
    <summary>CI Settings</summary>

**NOTE:** If your merge the PR with modified CI you **MUST KNOW** what you are doing
**NOTE:** Checked options will be applied if set before CI RunConfig/PrepareRunConfig step
- [ ]  Allow: Integration Tests
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Unit tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All with aarch64
- [ ]  Allow: All with ASAN
- [ ]  Allow: All with TSAN
- [ ]  Allow: All with Analyzer
- [ ]  Allow: All with Azure
- [ ]  Allow: Add your option here
---
- [ ]  Exclude: Fast test
- [ ]  Exclude: Integration Tests
- [ ]  Exclude: Stateless tests
- [ ]  Exclude: Stateful tests
- [ ]  Exclude: Performance tests
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN
- [ ]  Exclude: All with MSAN
- [ ]  Exclude: All with UBSAN
- [ ]  Exclude: All with Coverage
- [ ]  Exclude: All with Aarch64
---
- [ ]  do not test (only style check)
- [ ]  disable merge-commit (no merge from master before tests)
- [ ]  disable CI cache (job reuse)
- [ ]  allow: batch 1 for multi-batch jobs
- [ ]  allow: batch 2
- [ ]  allow: batch 3
- [ ]  allow: batch 4, 5 and 6
</details>

### CODE REVIEW

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit 97092e7f0cd1440f92152d858d7a72919f6e200b with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/64508/97092e7f0cd1440f92152d858d7a72919f6e200b/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>A Sync</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>⏳ pending</td></tr>
<tbody>
</table>
<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: docs/en/getting-started/install.md [hunk 1]
```diff
@@ -110,25 +110,6 @@ sudo service clickhouse-server start
...
-sudo apt-get update
-
-sudo apt-get install -y clickhouse-server clickhouse-client
-
-sudo service clickhouse-server start
-clickhouse-client # or "clickhouse-client --password" if you set up a password.
-```
-
-</details>
-
```
Comment (rschu1ze) [inline docs/en/getting-started/install.md:131]: Should we also get rid of section `Migration method for installing the deb-packages`? (l. 133ff)

File: docs/zh/getting-started/install.md [hunk 2]
```diff
@@ -38,26 +38,6 @@ sudo service clickhouse-server start
 clickhouse-client # or "clickhouse-client --password" if you've set up a password.
 ```
 
-<details markdown="1">
```
Comment (rschu1ze) [inline docs/zh/getting-started/install.md:41]: I like how the Chinese documentation is actually English 😄

File: docs/en/getting-started/install.md (hunk 1 above)
Comment (Felixoid) [reply docs/en/getting-started/install.md:131]: we can call it a new deprecated =)

It was a classic way to install keys, and old distros can still use it. Like, 14.04 or so

Comment (Felixoid) [reply docs/en/getting-started/install.md:131]: Done

**Review by rschu1ze — APPROVED** [review]

**Pull request merged**